docs/guide/installation/01_linux.md

```diff
@@ -29,17 +31,34 @@ lrwxrwxrwx. 1 root root 13 Oct 19 19:26 usb-Texas_Instruments_TI_CC2531_USB_CDC_
 ```
 
 ## Installing
+
+Procedure using curl and apt-get:
 ```bash
 # Set up Node.js repository and install Node.js + required dependencies
 # NOTE: Older i386 hardware can work with [unofficial-builds.nodejs.org](https://unofficial-builds.nodejs.org/download/release/v16.15.0/ e.g. Version 16.15.0 should work.
 sudo curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
 sudo apt-get install -y nodejs git make g++ gcc
+```
+
+Alternate procedure using snap store:
```

Comment:
This page assumes you are using a raspberry pi, I'm fine adding this but please add a note that this only works on ubuntu

Reply:
Relatively new to GIT edits but I'll try to add the note